src/cloud/project/project-patch.md

```diff
@@ -5,26 +5,20 @@ functional_areas:
   - Cloud
   - Upgrade
 ---
-The [{{site.data.var.mcp-prod}}](https://github.com/magento/magento-cloud-patches) package
-delivers Magento patches, which improve the integration of all `{{site.data.var.ee}}` versions with Cloud environments and supports quick delivery of critical and custom fixes.
 
-The {{ site.data.var.mcp }} package is a dependency for the {{site.data.var.ct}} package and is installed or updated when you install or update the {{ site.data.var.ct }} package version. You can also use and manage the {{ site.data.var.mcp-prod }} as a stand-alone package for an existing {{ site.data.var.ee }} project.
+The [Magento Quality Patches](https://github.com/magento/quality-patches) and [{{ site.data.var.mcp-prod }}](https://github.com/magento/magento-cloud-patches) packages deliver Magento [individual patches]({{ site.baseurl }}/release/policy/#individual-patch), which improve the integration of all {{site.data.var.ee}} versions with Cloud environments and supports quick delivery of critical and custom fixes. You can use these packages to apply, revert, and view general information about all individual patches that are available for Magento.
 
-The {{ site.data.var.mcp-prod }} package can deliver the following patch types:
+{:.bs-callout-tip}
+You can also use the [Magento Quality Patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html#mqp) and [{{ site.data.var.mcp-prod }}](#standalone) packages as stand-alone packages for {{ site.data.var.ce }} and {{ site.data.var.ee }} projects. We recommend using the Magento Quality Patches package for non-Cloud projects.
 
--  Required patches included in the {{ site.data.var.mcp-prod }} package
+When you deploy changes to the remote environment, `{{site.data.var.ct}}` uses `magento/quality-patches` and {{ site.data.var.mcp-package }} to check for pending patches and applies them automatically in the following order:
 
--  Optional Magento patches included in the `/m2-hotfixes` directory
-
--  Custom patches included in the `/m2-hotfixes` directory
-
-When you deploy changes to the remote environment, `{{site.data.var.ct}}` uses `{{site.data.var.mcp-package}}` to check for pending patches and applies them automatically in the following order:
-
-1. Apply required patches from Magento included in the `{{site.data.var.mcp-package}}` package.
+1. Apply required patches from Magento included in the {{ site.data.var.mcp-package }} package.
```

Comment:
```suggestion
1. Apply required patches from Magento included in the {{ site.data.var.mcp-prod }} package.
```

Use variable for "Magento Cloud Patches"  product name instead of GitHub package name to match the reference to Magento Quality Patches in the next step.